ui/components/src/Tooltip/index.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export * from './Tooltip';
+export * from './TooltipContent';
+export * from './tooltip-model';
+export * from './SeriesMarker';
+export * from './SeriesInfo';
+export * from './focused-series';
```

Comment:
Same question as before about whether all of these need to be exported as part of the public API?

Reply:
Nah they don't, I updated it so now only the Tooltip and SeriesMarker (could be used in the legend in the future) components are exported.